### ISSUE TITLE
False-positive for event scope

### ISSUE THREAD

**Issue by just-boris** [ilyavolodin/eslint-plugin-backbone#63]
Given the following code

```js
$(document).on('ajaxSend', someHandler).on('ajaxSuccess', anotherHandler);
```

It produces me `event-scope` error on the second `on` call. But there is no Backbone usage in this code, so error shouldn't appear there

**Comment by ilyavolodin** [comment]
Thanks for the issue. Interesting. I guess that's because of the chained call. Not really sure how I can fix this, right now, but I'll take a look.

**Comment by just-boris** [comment]
Also, I'd like to add, that it is not necessary to have several `on` calls.
This code also produces warning:

```js
$(document).doSomething().on();
```
It happens when you have `on` not the first in methods chain.
